src/module.js

```diff
@@ -777,6 +782,22 @@ export default class DailyIframe extends EventEmitter {
     this.sendMessageToCallMachine({ action: DAILY_METHOD_STOP_RECORDING });
   }
 
+  startLiveStreaming({ rtmpUrl,
```

Comment:
I haven't fully thought through how we want to standardize on return types, but it seems like `daily-js` methods generally returns one of three things:

- `this`, so that the method can be chained with others
- a promise with the result of the method (which it'd be possible to do here)
- nothing

Two things:

1. (Near-term consideration) Is a promise the most appropriate return type for this method?
2. (Longer-term consideration) It'd be great to eventually have a set of standards (return `this` for setup methods? return a promise for methods which could fail, so that consuming code could choose to use that promise rather than listen to another error event?)

Reply:
That's a really good point.

So, for this method, we'd return a Promise? (I actually thought I had done that, but clearly not, looking at the code here!)

Reply:
What's the right time to go through and change our method signatures? Is this something we should aim at doing to get to a 1.0?